app/services/fixed_charges/apply_taxes_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module FixedCharges
+  class ApplyTaxesService < BaseService
```

Comment:
can't we use `Charges::ApplyTaxesService.call(charge: fixed_charge, tax_codes:)` instead? 🤔